### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Fynkzia**
[DEMO LINK](https://Fynkzia.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by nikachu404 — APPROVED** [review]
Looks good)

File: src/App.tsx [hunk 1]
```diff
@@ -2,108 +2,37 @@ import React from 'react';
...
+function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => userId === user.id) || null;
+}
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+function getCommentById(id: number): Comment[] {
```
Comment (Ihor-Karpyn) [inline src/App.tsx:19]:
```suggestion
function getCommentsByPostId(postId: number): Comment[] {
```

File: src/components/PostInfo/PostInfo.tsx [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
```
Comment (Ihor-Karpyn) [inline src/components/PostInfo/PostInfo.tsx:29]: Wrap in span

File: src/components/CommentList/CommentList.tsx [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+  comments: Comment[],
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo
+        key={comment.id}
+        comment={comment}
+      />
```
Comment (Ihor-Karpyn) [inline src/components/CommentList/CommentList.tsx:16]: image

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 4]
```diff
@@ -2,108 +2,37 @@ import React from 'react';
...
+function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => userId === user.id) || null;
+}
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+function getCommentByPostId(postId: number): Comment[] {
```
Comment (Ihor-Karpyn) [inline src/App.tsx:19]:
```suggestion
function getCommentsByPostId(postId: number): Comment[] {
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**